src/loadContent.js

```diff
@@ -120,12 +120,18 @@ export function loadContent() {
     return;
   }
 
+  if (hashIds) {
+    learnerId = await hashString(learnerId);
+    learnerName = await hashString(learnerName);
+  }
+
   sourceUrlParser.search +=
     (sourceUrlParser.search.startsWith("?") ? "&" : "?") +
     "scorm" +
     `&learner_id=${learnerId}` +
     `&learner_name=${learnerName}` +
-    `&lms_origin=${encodeURIComponent(location.origin)}`;
+    `&lms_origin=${encodeURIComponent(location.origin)}` +
+    `&areIdsHashed=${hashIds}`;
```

Comment:
Given my previous comment I'd rename
```suggestion
    `&is_id_hashed=${hashIds}`;
```

(in fact the learnerName is not really an ID, so this variable name was a bit of a code smell to me)